test/bases/renku_data_services/data_api/test_sessions.py

```diff
@@ -451,6 +455,7 @@ async def test_patch_session_launcher(
     assert res.json.get("name") == patch_payload["name"]
     assert res.json.get("description") == patch_payload["description"]
     assert res.json.get("resource_class_id") == patch_payload["resource_class_id"]
+    assert res.json.get("disk_storage") == 3
```

Comment:
suggestion: We should add a test to check disk storage isn't modified if it isn't sent in the PATCH.

Reply:
I've added a commit with the test.

Reply:
:pray: Thank you!